development/kops/create_configuration.sh

```diff
@@ -53,9 +53,13 @@ echo "Creating cluster configuration"
 ${KOPS} create -f "./${KOPS_CLUSTER_NAME}/${KOPS_CLUSTER_NAME}.yaml"
 
 echo "Creating cluster ssh key"
-export SSH_KEY_PATH=${SSH_KEY_PATH:-$HOME/.ssh/id_rsa.pub}
-${KOPS} create secret --name $KOPS_CLUSTER_NAME sshpublickey admin -i ${SSH_KEY_PATH}
```

Comment:
This should not have been deleted The command is ${KOPS}

Reply:
Good catch, I still think we wanted to generate a random ssh key but I can make that change below